---
title: 57396 - ThrowingException
ms.custom: 
ms.date: 03/30/2017
ms.prod: .net-framework
ms.reviewer: 
ms.suite: 
ms.technology: dotnet-clr
ms.tgt_pltfrm: 
ms.topic: article
ms.assetid: 4f865de5-62c1-443c-9dcd-1675ca127334
caps.latest.revision: "3"
author: dotnet-bot
ms.author: dotnetcontent
manager: wpickett
ms.workload: dotnet
ms.openlocfilehash: 06378cb5d404830f210421091a1abaa0d7da2e3c
ms.sourcegitcommit: 16186c34a957fdd52e5db7294f291f7530ac9d24
ms.translationtype: MT
ms.contentlocale: ko-KR
ms.lasthandoff: 12/22/2017
---
# <a name="57396---throwingexception"></a>57396 - ThrowingException
## <a name="properties"></a>속성  
  
|||  
|-|-|  
|ID|57396|  
|키워드가|인프라|  
|수준|경고|  
|채널|Microsoft-Windows-응용 프로그램 서버-응용 프로그램/분석|  
  
## <a name="description"></a>설명  
 이 이벤트는 예외를 throw할 때 내보내집니다.  
  
## <a name="message"></a>메시지  
 예외를 throw하고 있습니다. 원본: %1.  
  
## <a name="details"></a>세부 정보
